client-sdk/ts-web/rt/package.json

```diff
@@ -16,6 +16,9 @@
     "dependencies": {
         "@oasisprotocol/client": "^0.1.0-alpha9",
         "elliptic": "^6.5.3",
+	"@noble/ed25519": "^1.6.0",
```

Comment:
we've been trying to use tweetnacl for ed25519 stuff. which when we last compared with noble's, tweetnacl was better audited. would it be possible to use that?